<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_vvod</class>
 <widget class="QWidget" name="Form_vvod">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Карточка</string>
  </property>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>120</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_vvod">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>50</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить и выйти</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>409</y>
     <width>1131</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Таблица  </string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1101</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>941</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Карточка задержаного</string>
   </property>
   <widget class="QComboBox" name="article_comboBox_num">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="article_comboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Статья наказания </string>
    </property>
   </widget>
   <widget class="QComboBox" name="number_comboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Вынесенное наказание</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fio_vvod">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите ФИО</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stamp_vvod">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите код отпечатка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Отпечаток</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fio">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия Имя Отчество</string>
    </property>
   </widget>
   <widget class="QComboBox" name="numberres_comboBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вынесенный  приговор</string>
    </property>
   </widget>
   <widget class="QComboBox" name="prigovor_comboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Текущая дата:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_v">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
